src/index.js

```diff
@@ -161,6 +161,54 @@ class EventPortal {
     }
   }
 
+  async getApplications(params=null) {
+    try {
+        console.log(`Fetching Applications`)
+        let url = `applications`
```

Comment:
For ease of collaboration and understanding, can we please change `url` var name to `endpoint` since the URL is constructed in the `api` function

Reply:
The only reason I included the ack and rejection action here is to demonstrate that the error queue (with custom name) receives the rejected events. I will remove this and just consume the events.